### PR TITLE
Spdm1.2: Add support to SessionPolicy and ContentChange

### LINKED ISSUE #231: Add support to SPDM 1.2 MEASUREMENTS response param2 ContentChange

### PR DESCRIPTION
**Pull request by longlongyang**
fix #231 and #232

Add the missing parts of Spdm1.2 new features MEASUREMENTS response param2
ContentChange and KEY_EXCHANGE request SessionPolicy.

Signed-off-by: Longlong Yang <longlong.yang@intel.com>

### CODE REVIEW

**Comment by jyao1** [comment]
please use MACRO for the new definition in SPDM

**Comment by longlongyang** [comment]
> please use MACRO for the new definition in SPDM

Updated.

File: spdmlib/src/message/mod.rs [hunk 1]
```diff
@@ -1112,6 +1114,7 @@ mod tests {
                     SpdmMeasurementSummaryHashType::SpdmMeasurementSummaryHashTypeNone,
                 slot_id: 100u8,
                 req_session_id: 100u16,
+                session_policy: 1,
```
Comment (jyao1) [inline spdmlib/src/message/mod.rs:1117]: please define this as MACRO as well.

File: spdmlib/src/common/mod.rs [hunk 2]
```diff
@@ -739,6 +740,7 @@ pub struct SpdmRuntimeInfo {
     pub message_c: ManagedBuffer,
     pub message_m: ManagedBuffer,
     pub message_vca: ManagedBuffer,
+    pub content_changed: bool, // used by responder, when content changed and spdm version is 1.2, set to true.
```
Comment (jyao1) [inline spdmlib/src/common/mod.rs:743]: we have 3 states: not capable, changed, not changed.

Comment (longlongyang) [reply spdmlib/src/common/mod.rs:745]: > we have 3 states: not capable, changed, not changed.

Updated. Still keep the type of content_changed in runtime info as bool, and changed the type of content_changed in SpdmMeasurementsResponsePayload struct to u8.

**Pull request merged**